entry_point.sh

```diff
@@ -0,0 +1 @@
+poetry run gunicorn "todo_app.app:create_app()" --bind 0.0.0.0:$PORT
```

Comment:
This relies on a PORT environment variable being set - we should either specify a default or update the Readme instructions to advise a developer they need to pass one in. Easiest might be to add a default to your dockerfile, e.g. with `ENV PORT=80` before the entrypoint